src/blocks/RecentScansBlock/Component.tsx

```
@@ -1,31 +1,51 @@
-      </div>
-
-      <div className="grid grid-cols-2 md:grid-cols-4 gap-6">
-        {[1, 2, 3, 4].map((id) => (
-          <div key={id} className="group cursor-pointer">
-            <div className="aspect-square overflow-hidden rounded-md bg-secondary/30 mb-2">
-              <img
-                src={`/placeholder.svg?height=300&width=300`}
-                alt={`Scan ${id}`}
-                className="h-full w-full object-cover transition-transform duration-300 group-hover:scale-105"
-              />
+      {false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)